src/vos/tests/vts_io.h

```diff
@@ -71,7 +71,7 @@ struct io_test_args {
 	const char		*dkey;
 	const char		*akey;
 	void			*custom;
-	int			 ofeat;
+	enum daos_otype_t 	 otype;
```

Comment:
```suggestion
	enum daos_otype_t	 otype;
```

Reply:
Looks like you have a space after daos_otype_t that can be removed.

Reply:
The space is kept there to be consistent with the following three lines. They all have a space to align with the above lines with a * sign before variable names.